src/System.Private.ServiceModel/tests/Common/Infrastructure/src/CertificateManager.cs

```diff
@@ -73,7 +73,7 @@ private static StoreLocation PlatformSpecificRootStoreLocation
                 using (X509Store store = new X509Store(storeName, storeLocation))
                 {
                     store.Open(OpenFlags.ReadOnly);
-                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint);
+                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint, validOnly: false);
```

Comment:
Validity could mean a lot of things... I tend to think that we should trust what the OS/underlying layers believe is valid or not as it helps us have a layer of security in addition to whatever checks we do here. 

I tend to take the more conservative approach here. If we leave this change in, could we add an issue to track flipping this back to `validOnly = true` ?